src/test/java/com/exasol/adapter/dialects/SqlGenerationVisitorTest.java

```diff
@@ -44,7 +44,7 @@ void testVisitSqlFunctionScalarCast() throws AdapterException {
         final DataType dataType = DataType.createDecimal(18, 0);
         final SqlLiteralDouble argument = new SqlLiteralDouble(20);
         final SqlFunctionScalarCast sqlFunctionScalarCast = new SqlFunctionScalarCast(dataType, argument);
-        assertThat(sqlGenerationVisitor.visit(sqlFunctionScalarCast), equalTo("CAST(20.0 AS DECIMAL(18, 0))"));
+        assertThat(sqlGenerationVisitor.visit(sqlFunctionScalarCast), equalTo("CAST(2E1 AS DECIMAL(18, 0))"));
```

Comment:
Is this required cause of https://github.com/exasol/virtual-schema-common-java/pull/210 ?

Reply:
Yes, we changed the format of double and exactnumeric